app/assets/stylesheets/me.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Me controller here.
```

Comment:
This file seems unnecessary

Reply:
Oops, will remove. was from the controller generator

Reply:
remove them!